xml/deployment_hwrecommend.xml

```diff
@@ -553,6 +553,12 @@
      </para>
     </listitem>
    </itemizedlist>
+     <note>
+       <para>
+         Ensure that OSD nodes do not have any networked block devices mapped,
```

Comment:
"Networked"? Would "network block devices" be enough?

Otherwise, fine!

Reply:
I did think about this. The block devices themselves are being put on a network (is my understanding), as opposed to the block devices being a network on a map. 
Although I think I may have read this sentence too many times.

Reply:
OK. I think just "network" is better TBH but it is not the end of the world.